### PR TITLE
Modèles d'impression

### PR DESCRIPTION
**Pull request by rbovard**
~Fix https://jira.liip.ch/browse/YC-1021~

### CODE REVIEW

**Comment by rbovard** [comment]
@AlexandreJunod As discussed I let you handle the date and page number I removed from default print template.

File: geocity/apps/reports/templates/reports/report.css [hunk 1]
```diff
@@ -1,16 +1,5 @@
...
-  @bottom-right-corner {
-    content: counter(page);
-    display: flex;
-    align-items: center;
-    justify-content: center;
-  }
-
-  @bottom-center {
-    content: "{% now "DATETIME_FORMAT" %}";
-  }
```
Comment (AlexandreJunod) [inline geocity/apps/reports/templates/reports/report.css:13]: Create block and find a way to add in python a counter, date time, and more.. https://www.quackit.com/css/at-rules/css_top-left-corner_at-rule.cfm (keep this in comment)

**Review by AlexandreJunod — CHANGES_REQUESTED** [review]

File: geocity/apps/reports/models.py [hunk 2]
```diff
@@ -42,6 +43,11 @@ class ReportLayout(models.Model):
             'La liste des polices disponibles est visible sur <a href="https://fonts.google.com/" target="_blank">Google Fonts</a>'
         ),
     )
+    font_size = models.PositiveIntegerField(
+        _("Taille de la police"),
+        default=12,
+        help_text=_("Taille de la police (en pixels). S'applique a tout le document"),
```
Comment (rbovard) [inline geocity/apps/reports/models.py:49]:
```suggestion
        help_text=_("Taille de la police (en pixels). S'applique à tout le document"),
```

File: geocity/apps/reports/migrations/0020_style_rename_font_reportlayout_font_family_and_more.py [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+            model_name="reportlayout",
+            old_name="font",
+            new_name="font_family",
+        ),
+        migrations.AddField(
+            model_name="reportlayout",
+            name="font_size",
+            field=models.PositiveIntegerField(
+                default=12,
+                help_text="Taille de la police (en pixels). S'applique a tout le document",
```
Comment (rbovard) [inline geocity/apps/reports/migrations/0020_style_rename_font_reportlayout_font_family_and_more.py:103]:
```suggestion
                help_text="Taille de la police (en pixels). S'applique à tout le document",
```

**Comment by AlexandreJunod** [comment]
Write test with a nice layout and more blocks

**Comment by AlexandreJunod** [comment]
Add Contact and details to show the sections correctly
Update author to show more informations
Add a planning to show dates
Add validations
Add somme amend properties

**Review by AlexandreJunod — APPROVED** [review]

**Pull request merged**